### PR TITLE
[3.0] Exact json utils

### PR DESCRIPTION
**Pull request by AlbumenJ**
## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/config/configcenter/DynamicConfiguration.java [hunk 1]
```diff
@@ -176,18 +174,6 @@ default boolean publishConfigCas(String key, String group, String content, Objec
...
-     * @param group the specified group
-     * @return the read-only non-null sorted {@link Set set} of config keys
-     * @throws UnsupportedOperationException If the under layer does not support
-     * @since 2.7.5
-     */
-    default SortedSet<String> getConfigKeys(String group) throws UnsupportedOperationException {
-        return Collections.emptySortedSet();
-    }
-
     /**
```
Comment (AlbumenJ) [inline dubbo-common/src/main/java/org/apache/dubbo/common/config/configcenter/DynamicConfiguration.java:177]: `getConfigKeys` is no longer needed.

**Comment by chickenlj** [comment]
related issue https://github.com/apache/dubbo/issues/10107

**Review by chickenlj — APPROVED** [review]
LGTM.

**Comment by chickenlj** [comment]
So what will be the side effects if users upgrade to the version with this change.

Will they need to add fastjson or gson dependencies explicitly to make their applications run?

**Comment by chickenlj** [comment]
> So what will be the side effects if users upgrade to the version with this change.
> 
> Will they need to add fastjson or gson dependencies explicitly to make their applications run?

Never Mind. I noticed the dependencies in dubbo-common/pom.xml and dubbo-all/pom.xml are not changed so no effects will be brought to end users.

File: dubbo-common/src/main/java/org/apache/dubbo/common/utils/JsonUtils.java [hunk 2]
```diff
@@ -16,12 +16,11 @@
...
-import com.google.gson.Gson;
+import org.apache.dubbo.common.json.JSON;
+import org.apache.dubbo.common.json.impl.FastJsonImpl;
 
 public class JsonUtils {
-    private static final Gson gson = new Gson();
-
-    public static Gson getGson() {
-        return gson;
+    public static JSON getJson() {
```
Comment (Shiyajian) [inline dubbo-common/src/main/java/org/apache/dubbo/common/utils/JsonUtils.java:30]: 为什么不是单例？

Comment (Shiyajian) [reply dubbo-common/src/main/java/org/apache/dubbo/common/utils/JsonUtils.java:30]: @chickenlj

Comment (AlbumenJ) [reply dubbo-common/src/main/java/org/apache/dubbo/common/utils/JsonUtils.java:30]: image
最新版本是单例的

**Pull request merged**